src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class FigureSupplier {
+    private static final Circle c = new Circle();
+    private static final IsoscelesTrapezoid it = new IsoscelesTrapezoid();
+    private static final Rectangle r = new Rectangle();
+    private static final RightTriangle rt = new RightTriangle();
+    private static final Square s = new Square();
+    private static final String[] figures =
+            {it.draw(), it.draw(), c.draw(), r.draw(), rt.draw(), s.draw()};
+
+    public static double getRandomProperties() {
+        double max = 1.0;
+        double min = 15.0;
+        return (int) ((Math.random() * (max - min)) + min);
+    }
+
+    public static String getRandomFigures() {
+        int min = 1;
+        int max = figures.length;
+        int randomNumber = (int) (Math.random() * (max - min)) + min;
+        StringBuilder sb = new StringBuilder();
+        while (randomNumber != 0 && randomNumber <= figures.length) {
+            sb.append(figures[randomNumber]).append("\n");
+            randomNumber--;
+        }
+        return sb.toString();
+    }
```

Comment:
I made getRandomFigure() but I don't understand why I should do a different method for every child where I need something like radius, perimeter, or height (getRandomCircle, getRandomPArametre)? I achieved it with add to abstract parent class appropriate getter already with a random color.
And about create the method getRandomCircle or getRandomSquare I thought it doesn't matter because the method getRandomFigure does it also.